### DISCUSSION TITLE
Run the “from towhee import pipe ” command，an error occurred：AttributeError: module 'towhee' has no attribute 'runtime'

### DISCUSSION THREAD

**Discussion by Fmmmmmmmmm** [towhee-io/towhee#2668] [Q&A]
For help, I installed wowhee using pip3 in python3.6, and towhee was displayed on the pip3 list. Then I ran "from towhee import pipe" on the python3 command line, and an error was displayed as "AttributeError: module 'towhee' has no attribute 'runtime' ", what should I do?
![image](https://github.com/towhee-io/towhee/assets/125537828/cc6c57d0-3fd4-49c4-adb5-bde85e0c41be)

**Comment by junjiejiangjjj** [comment]
towhee only supports >=python3.7
